### PR TITLE
Fix: preset modules selected and deselected

### LINKED ISSUE #10660: Preset: modules selected and deselected
Strange behavior. The initial modules are selected or deselected when switching between presets

To reproduce the issue:
- go to https://lite.jhipster.tech/landscape
- select 1st preset
  - the module `application-service-hexagonal-architecture-documentation` is selected
- select the 2nd preset
  - the module `application-service-hexagonal-architecture-documentation` is not selected anymore, but in config it is

### PR DESCRIPTION
**Pull request by qmonmert**
Fix #10660

### CODE REVIEW

File: src/main/webapp/app/module/primary/landscape/Landscape.component.ts [hunk 1]
```diff
@@ -411,6 +411,12 @@ export default defineComponent({
         return;
       }
 
+      landscapeValue()
+        .selectedModules()
+        .forEach(module => {
+          landscape.value.loaded(landscapeValue().toggle(module));
+        });
+
```
Comment (renanfranca) [inline src/main/webapp/app/module/primary/landscape/Landscape.component.ts:419]: @qmonmert, if you could create a test for this, it would be really nice 👌

**Review by renanfranca — APPROVED** [review]
Thanks for your work @qmonmert ! 😊👍

File: src/main/webapp/app/module/primary/landscape/Landscape.component.ts (hunk 1 above)
Comment (pascalgrimaud) [reply src/main/webapp/app/module/primary/landscape/Landscape.component.ts:419]: yes, there should be a test to prove our initial code was with a bug

**Comment by qmonmert** [comment]
@renanfranca @pascalgrimaud 
image

File: src/test/webapp/unit/module/primary/landscape/LandscapeComponent.spec.ts [hunk 2]
```diff
@@ -1119,6 +1119,22 @@ describe('Landscape', () => {
...
+      });
+
+      ({ initModuleElement } = await setupAndSelectPreset('init-typescript'));
+
+      classes = initModuleElement.classes();
+      ['-selected', '-compacted'].forEach(expectedClass => {
+        expect(classes).toContain(expectedClass);
+      });
+    });
+
```
Comment (renanfranca) [inline src/test/webapp/unit/module/primary/landscape/LandscapeComponent.spec.ts:1137]: 👍👍👍👌

**Review by renanfranca — APPROVED** [review]
Thanks @qmonmert 👌

**Pull request merged**